// react-native-apple-header/build/dist/AppleHeader.style.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports._largeTitleStyle = exports._dateTitleStyle = exports.container = void 0;
const react_native_1 = require("react-native");
exports.container = (backgroundColor, borderColor) => {
    return {
        borderColor,
        backgroundColor,
        paddingBottom: 8,
        flexDirection: "row",
        marginHorizontal: 16,
        borderBottomWidth: 1,
        alignItems: "flex-start",
        justifyContent: "space-between",
    };
};
exports._dateTitleStyle = (fontColor, fontSize, fontWeight) => {
    return {
        fontSize,
        fontWeight,
        lineHeight: 18,
        color: fontColor,
        letterSpacing: react_native_1.Platform.OS === "ios" ? -0.08 : undefined,
    };
};
exports._largeTitleStyle = (fontColor, fontSize, fontWeight) => {
    return {
        fontSize,
        fontWeight,
        lineHeight: 41,
        color: fontColor,
        letterSpacing: react_native_1.Platform.OS === "ios" ? 0.41 : undefined,
    };
};
exports.default = react_native_1.StyleSheet.create({
    avatar: {
        height: 43,
        width: 43,
        borderRadius: 43 / 2,
    },
    avatarContainerStyle: {
        alignSelf: "center",
        justifyContent: "center",
    },
});
