// react-native-apple-header/build/dist/AppleHeader.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const tslib_1 = require("tslib");
const React = tslib_1.__importStar(require("react"));
const react_native_1 = require("react-native");
const AppleHeader_style_1 = tslib_1.__importStar(require("./AppleHeader.style"));
const AppleHeader = (props) => {
    const { onPress, dateTitle, largeTitle, avatarStyle, imageSource, containerStyle, dateTitleStyle, largeTitleStyle, borderColor, backgroundColor, dateTitleFontSize, dateTitleFontColor, dateTitleFontWeight, largeTitleFontSize, largeTitleFontColor, largeTitleFontWeight, } = props;
    return (<react_native_1.View style={containerStyle || AppleHeader_style_1.container(backgroundColor, borderColor)}>
      <react_native_1.View>
        <react_native_1.Text style={dateTitleStyle ||
        AppleHeader_style_1._dateTitleStyle(dateTitleFontColor, dateTitleFontSize, dateTitleFontWeight)}>
          {dateTitle}
        </react_native_1.Text>
        <react_native_1.Text style={largeTitleStyle ||
        AppleHeader_style_1._largeTitleStyle(largeTitleFontColor, largeTitleFontSize, largeTitleFontWeight)}>
          {largeTitle}
        </react_native_1.Text>
      </react_native_1.View>
      <react_native_1.TouchableOpacity style={AppleHeader_style_1.default.avatarContainerStyle} onPress={onPress}>
        <react_native_1.Image style={avatarStyle} source={imageSource} {...props}/>
      </react_native_1.TouchableOpacity>
    </react_native_1.View>);
};
AppleHeader.defaultProps = {
    dateTitleFontSize: 13,
    largeTitle: "For You",
    dateTitleFontWeight: "600",
    largeTitleFontSize: 34,
    borderColor: "#EFEFF4",
    dateTitleFontColor: "#8E8E93",
    avatarStyle: AppleHeader_style_1.default.avatar,
    largeTitleFontWeight: "bold",
    backgroundColor: "transparent",
    dateTitle: "MONDAY, 27 NOVEMBER",
};
exports.default = AppleHeader;
